common/grades.py

```diff
@@ -208,8 +220,11 @@ def print_submission_grades(
             all_comments.insert(0, "(LATE)")
         concatted_comments = "; ".join(all_comments)
 
-        if rubric_code == "ALL" and self.is_late(name):
-            total_pts = round(total_pts * (1 - LATE_PENALTY), 2)
+        late_penalty = DEFAULT_LATE_PENALTY
+        if "late_penalty" in self.rubric:
+            late_penalty = self.rubric["late_penalty"]
+        if rubric_code == "ALL" and self.is_late(name) and late_penalty != 0:
+            total_pts = round(total_pts * (1 - late_penalty), 2)
```

Comment:
i agree with this for the future, i was thinking we should at least add an additional option for flat point deductions (especially doable since now it's specified per-assignment) but this is probably a better idea